yt_dlp/extractor/crunchyroll.py

```diff
@@ -359,18 +352,12 @@ def entries():
         else:
             raise ExtractorError(f'Unknown object type {object_type}')
 
-        # There might be multiple audio languages for one object (`<object>_metadata.versions`),
-        # so we need to get the id from `streams_link` instead or we dont know which language to choose
-        streams_link = response.get('streams_link')
-        if not streams_link and traverse_obj(response, (f'{object_type}_metadata', 'is_premium_only')):
+        if traverse_obj(response, (f'{object_type}_metadata', 'is_premium_only')) and not self.is_premium:
```

Comment:
When trying to download a concert or music video with an account that doesn't have premium, `streams_link` is populated and thus the check doesn't work.
For normal episodes it still works as you said, but I wanted to have one solution single for everything instead of two

Reply:
I would be careful with this check for episodes, as I said there was problems with false positives. If the check cannot be used for music then ill have to re-review for the token code.

Reply:
I've used this check in another project for some time an I never had an issue so far. Let's hope that the false positives you described are fixed

Reply:
I think we can trust the results from crunchy-cli 🙏